data/presets/traffic_sign/hazard/horse_riders.json

```
@@ -0,0 +1,19 @@
+{
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "traffic_sign": "hazard",
+        "hazard": "horse_riders"
+    },
+    "reference": {
+        "key": "hazard",
+        "value": "horse_riders"
+    },
+    "aliases": [
+        "Equestrian Rider Sign",
```

An equestrian is a rider:

```suggestion
        "Equestrian Sign",
```